cmd/platform/server.go

```diff
@@ -76,8 +76,18 @@ func runServer(configFileLocation string) {
 		a.LoadLicense()
 	}
 
+	a.InitBuiltInPlugins()
+
```

Comment:
unrelated to your PR, but `model.CLIENT_DIR` isn't very descriptive.  I assume this was intended in the `FindDir`?